docs/source/contribution/development_for_databricks.md

```diff
@@ -0,0 +1,89 @@
+
+To make developing Kedro for Databricks easier, Kedro comes with a `Makefile` target named `databricks-build` that automates the process of building a wheel file and installing this on your Databricks cluster to save development time.
+
+### Setup the Databricks CLI to test a Kedro build
+
+Before you use `make databricks-build`, you must [set up the Databricks CLI](https://docs.databricks.com/dev-tools/cli/index.html#set-up-the-cli).
+
+Next, create and environment variable with the ID of the cluster you are using to test your Kedro build. You can find the ID by executing the Databricks CLI command `databricks clusters list` and looking for the Cluster ID to the left of the name of your chosen cluster, for instance:
+
+```bash
+$ databricks clusters list
+1234-567890-abcd1234  General Cluster             TERMINATED
+0987-654321-9876xywz  Kedro Test Cluster          TERMINATED
+```
+
+In this case, the cluster ID of `Kedro Test Cluster` is `0987-654321-9876xywz`.
+
+Once you have determined the cluster ID, you must export it to an environment variable named `DATABRICKS_CLUSTER_ID`:
+
+```bash
+# Linux or macOS
+export DATABRICKS_CLUSTER_ID=<your-cluster-id>
+
+# Windows
+set DATABRICKS_CLUSTER_ID=<your-cluster-id>
+```
+
+### How to use `make databricks-build` to test your Kedro build
+
+With the setup complete, you can use `make databricks-build`. In your terminal, navigate to the parent directory of your Kedro development repository and run:
+
+```bash
+make databricks-build
+```
+
+You should see a stream of messages being written to your terminal. Behind the scenes, `build-databricks` does the following:
+
+1. Builds a wheel file of your modified version of Kedro.
+2. Uninstalls any library on your Databricks cluster with the same wheel file name.
+3. Uploads your updated wheel file to DBFS (Databricks File System).
+4. Queues your updated wheel file for installation
+5. Restarts your cluster to apply the changes.
+
+Note that your cluster will be unavailable while it restarts. You can poll the status of the cluster using the Databricks CLI command `databricks clusters get --cluster-id <your-cluster-id> | grep state` (macOS, Linux).
+
+Once the cluster has restarted, you should verify that your modified version of Kedro has been correctly installed. Run `databricks libraries list --cluster-id <your-cluster-id>`. If installation was successful, you should see the following output:
+
+```bash
+{
+  "cluster_id": "<your-cluster-id>",
+  "library_statuses": [
+    {
+      "library": {
+        "whl": "dbfs:/tmp/kedro-builds/kedro-<version>-py3-none-any.whl"
+      },
+      "status": "INSTALLED",
+      "is_library_for_all_clusters": false
+    }
+  ]
+}
+```
+
+Any runs of a Kedro project on this cluster will now reflect your latest local changes to Kedro. You can now test your changes to Kedro by using your cluster to run a Kedro project.
```

Comment:
I'm guessing that with any new change you'd have to run `make databricks-build` again?

Reply:
That's right, this is a one-off run. The alternative would be to create a process which watches `/dist`, using e.g. `dbx sync`.

Reply:
I think this makes sense for now. A continues sync process might be handy if we're building the kedro-databricks plugin, but by that time there might already be another way to do that.